README.md

```diff
@@ -18,15 +19,16 @@ Note that `mapbox-vector-tile` depends on [Shapely](https://pypi.python.org/pypi
 Encoding
 --------
 
-Encode method expects an array of layers or atleast a single valid layer. A valid layer is a dictionary with the following keys
+Encode method expects an array of layers or at least a single valid layer. A valid layer is a dictionary with the
+following keys
 
 * `name`: layer name
 * `features`: an array of features. A feature is a dictionary with the following keys:
 
   * `geometry`: representation of the feature geometry in WKT, WKB, or a shapely geometry. Coordinates are relative to the tile, scaled in the range `[0, 4096)`. See below for example code to perform the necessary transformation. *Note* that `GeometryCollection` types are not supported, and will trigger a `ValueError`.
   * `properties`: a dictionary with a few keys and their corresponding values.
 
-```python
+```pycon
```

Comment:
what does this do differently?

Reply:
Please excuse me. It was for my IDE when I did automatic replacement (eg. `addFeatures` ⇾ `add_features`). I will revert these changes in the README file.